domains/vanagas.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "tomasvanagas",
+           "email": "tomasvanagas@ymail.com",
+           "discord": "693891862519480351"
+        },
+    
+        "record": {
+            "A": ["158.129.172.231"]
```

Comment:
Page isn't working, please give a preview of your website

Reply:
I will be hosting my own nextcloud server on this domain. Currently I only configured a test html:

```bash
curl http://vanagas.is-a.dev
<center>Hello World from vanagas.is-a.dev domain name</center>%  
```